mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go

```diff
@@ -0,0 +1,245 @@
+package mongodbatlas
+
```

Comment:
@martinstibbe do we need a _test.go as part of this new supporting resource?

Reply:
+1